scripts/openstack.py

```diff
@@ -15,7 +15,13 @@ def get_key_parser():
     )
     parser.add_argument(
         '--key-filename',
-        help='path to the ssh private key',
+        help='path to the ssh private key. Default: %(default)s',
+        action='append',
```

Comment:
I'm not sure the feature will work as expected? Do you think `action='append'` will put given parameter to the top of default?
How is your `--help` output look like for this command?

Reply:
you are right - `append` does it a) at the end b) still silently tries the default keys. fixed now. The output of help is:
```
  --key-filename KEY_FILENAME
                        path to the ssh private key. Default:
                        ['/home/xxx/.ssh/id_rsa', '/home/xxx/.ssh/id_dsa',
                        '/home/xxx/.ssh/id_ecdsa']
```

Reply:
Fixed now. The behavior is now:
- if no `--key-filename` is given, the default keys will be used
- If `--key-filename` is given, only that key is used

Reply:
Resolving the conversation and assuming this is done in the current code.